libraries/stdlib/py/runtime/arrays.kt

```diff
@@ -7,13 +7,21 @@ package kotlin.js
 
 import withType
 
+private external fun <T> list(): Array<T>
+
 @PublishedApi
-internal external fun <T> Array(size: Int): Array<T>
+internal fun <T> Array(size: Int): Array<T> {
+    val result = list<T>()
+    repeat(size) {
+        result.asDynamic().append(null)
+    }
+    return result
+}
 
 @PublishedApi
 internal fun <T> fillArrayVal(array: Array<T>, initValue: T): Array<T> {
-    for (i in 0..array.size - 1) {
-        array[i] = initValue
+    repeat(array.size) {
+        array[it] = initValue
```

Comment:
This is just a cosmetic change, not actually required. If you want, I can extract it to a separate commit.

Reply:
Yeah, a separate commit (the same PR) would be nice, to reduce diff noise.